catalog/resource_sql_table.go

```diff
@@ -29,14 +33,19 @@ type SqlTableInfo struct {
 	TableType             string            `json:"table_type" tf:"force_new"`
 	DataSourceFormat      string            `json:"data_source_format,omitempty" tf:"force_new"`
 	ColumnInfos           []SqlColumnInfo   `json:"columns,omitempty" tf:"alias:column,computed,force_new"`
+	Partitions            []string          `json:"partitions,omitempty" tf:"force_new"`
+	ClusterKeys           []string          `json:"cluster_keys,omitempty" tf:"force_new"`
```

Comment:
I'm not sure about the `force_new` - you can change cluster keys without dropping a table.